services/src/contexts/migrations/migration_0003_gbif_config.rs

```diff
@@ -106,11 +102,13 @@ mod tests {
         // drop the connection because the pool is limited to one connection, s.t. we can reuse the temporary schema
         drop(conn);
 
+        /* Note: this only works if the rust type is
```

Comment:
either remove it or write some SQL that checks the provider survived the migration

Reply:
[c3fe060](https://github.com/geo-engine/geoengine/pull/920/commits/c3fe060c999ac8b2bb6d30dbd1496247b74dff0e)